scripts/npm/unlink/index.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var path = require( 'path' );
+var fs = require( 'fs' );
+var componentsDir = path.join( __dirname, '..', '..', '..', 'components' );
+var components = [ 'cf-buttons',
```

Comment:
I wonder if we could store the list of components in a single place, rather than in each of these scripts. A custom property in `package.json`, perhaps? Not a blocker, though.